### PR TITLE
update: explanation + example for bit-wise

### PR DESCRIPTION
**Pull request by mattiasduerrmeier**
Adding explanation + example for the bit-wise AND, OR and XOR could make it easier to understand them.
The bit-wise invert has a link for more details. I left it as it is.

### CODE REVIEW

File: op_exp.md [hunk 1]
```diff
@@ -61,16 +61,16 @@ Here is a quick overview of the available operators:
...
-    - `5 | 3` gives `7`
-
-- `^` (bit-wise XOR)
-    - Bitwise XOR of the numbers
-    - `5 ^ 3` gives `6`
+    - Bitwise OR of the numbers: if both bits are `0`, the result is `0`. Else it's `1`. 
+    - `5 | 3` gives `7` (`0101 | 0011` gives `0111`)
+    
+- `^` (bit-wise XOR) 
+    - Bitwise XOR of the numbers: if both bits (`1 or 0`) are the the same, the result is `0`. Else it's `1`.
```
Comment (swaroopch) [inline op_exp.md:72]: @mattiasduerrmeier minor nit - can we please remove duplicate word : `the the` -> `the`

Overall, LGTM!

Comment (mattiasduerrmeier) [reply op_exp.md]: Done !

**Comment by mattiasduerrmeier** [comment]
I see that the pull request is still open.

To make sure you know I corrected the small typos, I'm writing another comment so that you can merge it.

**Comment by swaroopch** [comment]
Thank you!

**Pull request merged**